pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqluser/_vcr_cassettes/tf.yaml

```diff
@@ -44,29 +44,29 @@ interactions:
       body: fake error message
       headers:
         Content-Type:
-          - application/json; charset=UTF-8
+            - application/json; charset=UTF-8
       status: 404 Not Found
       code: 404
-      duration: 181.438833ms
+      duration: 190.777574ms
   - id: 1
     request:
       proto: HTTP/1.1
       proto_major: 1
       proto_minor: 1
-      content_length: 316
+      content_length: 339
       transfer_encoding: []
       trailer: {}
       host: sqladmin.googleapis.com
       remote_addr: ""
       request_uri: ""
       body: |
-        {"databaseVersion":"MYSQL_5_7","name":"sqluser-dep-3vxr1tn21m3yp","region":"us-central1","settings":{"activationPolicy":"ALWAYS","availabilityType":"ZONAL","dataDiskType":"PD_SSD","pricingPlan":"PER_USE","storageAutoResize":true,"tier":"db-n1-standard-1","userLabels":{"cnrm-test":"true","managed-by-cnrm":"true"}}}
+          {"databaseVersion":"MYSQL_5_7","name":"sqluser-dep-3vxr1tn21m3yp","region":"us-central1","settings":{"activationPolicy":"ALWAYS","availabilityType":"ZONAL","dataDiskType":"PD_SSD","edition":"ENTERPRISE","pricingPlan":"PER_USE","storageAutoResize":true,"tier":"db-n1-standard-1","userLabels":{"cnrm-test":"true","managed-by-cnrm":"true"}}}
```

Comment:
This could be the only reason that caused the previous test to fail. The other updates are all in responses; we do not worry about response content with replay mode, but only check the request body, URL, and method.

Reply:
that's fair

Reply:
Noticed that after SQL Instance is switched from TF to direct controller, the sqlinstance related requests in the old vcr `tf.yaml` is invalid. We do need to re-collect logs and save the direct log(and DCL log) in a new folder `nontf.yaml`.